README.md

```diff
@@ -17,6 +17,7 @@ Previously, the website and documentation for the Draft U.S. Web Design Standard
   * [Download](#download)
   * [Install using npm](#install-using-npm)
   * [Use another framework or package manager](#use-another-framework-or-package-manager)
+  * [Use the latest version of the `uswds` package](#use-the-latest-version-of-the-uswds-package)
```

Comment:
So I'm not sure if this is needed, or if it is maybe it should be stated differently bc it deals with utilizing the docs repo. Linking to that repo by stating something along the lines of "run the docs site" and linking to that repo may be enough.